### ISSUE TITLE
Finding the certificate to uninstall it from a bad install

### ISSUE THREAD

**Issue by LavenderTheGreat** [felixse/FluentTerminal#529]
When installing this I accidentally chose to install into the automatically determined store on my machine, thing is, I have no idea where this is or what the Certificate's internal names are to uninstall it from wherever it installed to.

I doubt there's any harm in it being there (since it's made for this app and nothing more afaik) but it'd be nice to not have it there to keep the certificate stores clean.

TLDR: What should I be searching for in the search menus to find the certificate to uninstall them if I installed it in the wrong place?

**Comment by felixse** [comment]
Search `Manage User Certificates` in the start menu and open it.  The automatically installed certificate will be in `Intermediate Certification Authorities -> Certificates` and is named `BDC72197-B634-438A-B863-16DCB7D90C85`
